pyproject.toml

```diff
@@ -0,0 +1,97 @@
+[tool.black]
+line-length=119
+target-version=["py38"]
+
+[tool.isort]
+atomic=true
+line_length = 119
+lines_after_imports=2
+lines_between_types=1
+use_parentheses = true
+balanced_wrapping = true
+include_trailing_comma = true
+multi_line_output = 3
+known_third_party=[
+    "pytest",
+]
+known_first_party=["convert2rhel"]
+
+[tool.coverage.run]
+branch = false
+source = ["convert2rhel"]
+omit = ["convert2rhel/unit_tests"]
+
+[tool.coverage.paths]
+source = ["convert2rhel/"]
+
+[tool.coverage.report]
+fail_under = 87
```

Comment:
Last week we moved it up to 80. Are you changing the coverage check so that it reports 87% now?

Reply:
you're right, made a little mistake in migrating the config. Thx